packages/react-test/src/App.tsx

```diff
@@ -202,21 +203,22 @@ function App() {
               <h3>Test your component here</h3>
               {/* Normal version */}
               <div>
-                <Accordion
-                  isOpen={true}
-                  isFullWidth
-                  openLabel={'Show'}
-                  closeLabel={'Hide'}
-                  content={
-                    <Accordion
-                      isOpen={true}
-                      isFullWidth
-                      openLabel={'Show'}
-                      closeLabel={'Hide'}
-                      content={<div>Content</div>}
-                    />
-                  }
-                />
+                <EDropdownComponent
+                  label="Label"
+                  placeholder="Plassholder"
+                  onValueChange={(e) => {
+                    console.log('Value changed:', e.detail.value);
+                  }}
+                >
+                  {[...Array(10)].map((_, i) => (
+                    <EOptionComponent key={i} value={i.toString()}>
+                      <div className="e-flex e-gap-8 e-w-100" style={{ justifyContent: 'space-between' }}>
```

Comment:
Tiny thing, but this is probably something we could do through the styles on the `EOptionComponent`